apps/studio/components/interfaces/Support/Success.tsx

```diff
@@ -29,8 +38,15 @@ const Success = ({ sentCategory = '' }: SuccessProps) => {
       <div className="flex items-center flex-col space-y-2">
         <h3 className="text-xl">Support request successfully sent!</h3>
         <p className="text-sm text-foreground-light">
-          We will reach out to you at <span className="text-foreground">{respondToEmail}</span>
+          We will reach out to you at <span className="text-foreground">{respondToEmail}</span>.
         </p>
+        {selectedProject !== 'no-project' && (
+          <p className="text-sm text-foreground-light">
+            Your ticket has been logged for the project{' '}
```

Comment:
Given the customer just selected this, not sure if we're adding any value by repeating what they have selected?

If we add value, project name should be enough, no need to add a ref id or anything

Reply:
Is this even something you changed? Seems kinda unrelated and might just wrongfully show up here?

Reply:
No, that's not my changes.